internal/integration/testdata/sqlite/index-desc.txt

```diff
@@ -0,0 +1,22 @@
+apply 1.hcl
+cmpshow users 1.sql
+
+-- 1.hcl --
+schema "main" {}
+
+table "users" {
+    schema = schema.main
+    column "rank" {
+        type = int
+    }
+    index "rank_idx" {
+        on {
+            desc   = true
+            column = table.users.column.rank
+        }
+    }
+}
```

Comment:
Shouldn't you add a test for `columns` too? (Check, there are no changes proposed by the planner).

Reply:
Next pr. I tried to make it an "initial infrastructure" for SQLite testing.